test/runtests.jl

```diff
@@ -295,6 +299,23 @@ end
     end
 end
 
+@testset "Hierarchy" begin
+    Ps = SemiclassicalJacobi.(2, 0:100,0,0)
```

Comment:
@TSGut @MikaelSlevinsky This is really nice: we can construct a whole family of semiclassical OPs, getting say the first 100 in less than a second. And amazingly it's stable. 

(Someone PLEASE stop me before I try to make `SemiclassicalJacobi.(2, a, b, 0:∞)` also work 🤪)

Reply:
I've had a chance to go through this now, it looks really nice. It looks like it wouldn't be too much work to change the structure of the stuff in https://github.com/JuliaApproximation/SemiclassicalOrthogonalPolynomials.jl/pull/18 to fit in with this, let me know when and what you might need for that purpose.

Reply:
Yes I think the only big change is now we store the Jacobi operator (instead of a pointer to another OP basis).